cluster/ci/origin/pipelines/build/Jenkinsfile

```diff
@@ -11,11 +11,19 @@ pipeline {
   agent { label "release-ci.jenkins-slave" }
 
   parameters {
-    string(defaultValue: "${env.ORIGIN_TARGET_BRANCH}", name: "ORIGIN_TARGET_BRANCH", description: "The branch in the origin repository to test against.")
-    string(defaultValue: "${env.PULL_REFS}", name: "PULL_REFS", description: "Used by prow. If this is not a pull request, it should at least contain the branch:hash of the HEAD being tested.")
-    string(defaultValue: "${env.BUILD_ID}", name: "buildId", description: "The ID that prow sets on a Jenkins job in order to correlate it with a ProwJob")
-    string(defaultValue: "${env.ORIGIN_URL}", name: "ORIGIN_URL", description: "The URL of the Origin repository to use")
+    string(defaultValue: "${env.ORIGIN_URL ?: 'https://github.com/openshift/origin.git' }", name: "ORIGIN_URL", description: "The URL of the Origin repository to use")
     booleanParam(defaultValue: (env.BUILD_IMAGES == "true" || env.BUILD_IMAGES == "yes"), name: "BUILD_IMAGES", description: "Set to 'true' if you wish to build images")
+    string(defaultValue: "${env.BUILD_ID}", name: "buildId", description: "The ID that prow sets on a Jenkins job in order to correlate it with a ProwJob")
+
+    // Only PULL_REFS is currently used, but other parameters are here for completeness
+    // and future logging
+    string(name: "REPO_OWNER", defaultValue:"${env.REPO_OWNER ?: 'openshift'}")
```

Comment:
Defaulting these makes it impossible to tell if you are a periodic job

Reply:
So I split these off from other PULL variables. So if you have these but not PULL_REFS, then you do have a periodic job. 
Even a periodic job ... wouldn't it make sense that it be associated with a repo name, owner ?

Reply:
Not necessarily -- jobs to monitor resource usage, janitors, re-tester bot are all periodics unrelated to any repo. I agree periodics tied to a repo are a good thing and we should try to invent a new class of job ... but right now the spec says no

https://github.com/openshift/test-infra/tree/master/prow#job-evironment-variables

Reply:
ok, will revert the split for now